### PR TITLE
chore: Add `@MetaMask/metamask-assets` to `CODEOWNERS`

### PR DESCRIPTION
**Pull request by gambinish**
## **Description**

Initial pass at adding certain codepaths for [assets team](https://github.com/orgs/MetaMask/teams/metamask-assets) to codeown. Feel free to suggest additions or removals for things I don't have context about!

## **Related issues**

Fixes: https://consensyssoftware.atlassian.net/browse/MMASSETS-391

## **Manual testing steps**

N/A

## **Screenshots/Recordings**

N/A

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -63,3 +63,32 @@ patches/react-native+0.*.patch       @MetaMask/supply-chain
...
+app/components/Views/Asset @MetaMask/metamask-assets
+app/components/Views/AssetDetails @MetaMask/metamask-assets
+app/components/Views/AssetHideConfirmation @MetaMask/metamask-assets
+app/components/Views/AssetOptions @MetaMask/metamask-assets
+app/components/Views/Collectible @MetaMask/metamask-assets
+app/components/Views/CollectibleView @MetaMask/metamask-assets
+app/components/Views/DetectedTokens @MetaMask/metamask-assets
+app/components/Views/NFTAutoDetectionModal @MetaMask/metamask-assets
+app/components/Views/NftDetails @MetaMask/metamask-assets
+app/reducers/collectibles @MetaMask/metamask-assets
```
Comment (sethkfman) [inline .github/CODEOWNERS:94]:
```suggestion
app/reducers/collectibles @MetaMask/metamask-assets

```

**Review by NicolasMassart — COMMENTED** [review]
Suggestion: Would it be possible to group files under a common folder, for instance app/components/UI/assets for the UI so that you could apply the owners only to the folder? Having it applied on each individual file is going to make it had for any refactor if needed and would force to add owner line each time you add a new one.

**Comment by gambinish** [comment]
> Suggestion: Would it be possible to group files under a common folder, for instance app/components/UI/assets for the UI so that you could apply the owners only to the folder? Having it applied on each individual file is going to make it had for any refactor if needed and would force to add owner line each time you add a new one.

This was something that we talked about during our initial discovery call. I think that this is where we would like to end up, but that we decided that separating by file (the changes in this PR) would be the first step along that effort.

Going to tag @sethkfman  and @Cal-L on this comment for visibility. Should we handle the `assets` subdirectory here or does it make sense to save that for later?

**Comment by sethkfman** [comment]
@NicolasMassart Great callout. This will be the next phase of CODEOWNER implementation.

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**